test/parallel/test-internal-modules.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
-var common = require('../common');
-var path = require('path');
-var assert = require('assert');
+const common = require('../common');
+const path = require('path');
+const assert = require('assert');
 
-assert.throws(function() {
+assert.throws(function(err, cb) {
+  if (err) return cb(err);
   require('internal/freelist');
 });
```

Comment:
Ping @christyleung1987: Still working on this? (If you have any questions or need additional assistance, feel free to ask here or on the #node-dev channel on Freenode IRC.)